test/images/agnhost/webhook/pods.go

```diff
@@ -77,6 +80,25 @@ func admitPods(ar v1.AdmissionReview) *v1.AdmissionResponse {
 }
 
 func mutatePods(ar v1.AdmissionReview) *v1.AdmissionResponse {
+	selector := func(pod *corev1.Pod) bool {
+		return pod.Name == "webhook-to-be-mutated"
```

Comment:
I expected this to return false if the pod already had an init container named `webhook-added-init-container`

Reply:
Done.